sdk/monitor/monitor-opentelemetry-exporter/src/utils/common.ts

```diff
@@ -287,3 +287,10 @@ export function serializeAttribute(value: AnyValue): string {
 export function shouldCreateResourceMetric(): boolean {
   return !(process.env[ENV_OPENTELEMETRY_RESOURCE_METRIC_DISABLED]?.toLowerCase() === "true");
 }
+
+export function isSyntheticSource(attributes: Attributes): boolean {
+  const syntheticType: string = attributes[
+    experimentalOpenTelemetryValues.SYNTHETIC_TYPE
+  ] as string;
+  return Boolean(syntheticType);
```

Comment:
If we are not checking for whitespace content then this can be simplified to

```ts
    return !!attributes[experimentalOpenTelemetryValues.SYNTHETIC_TYPE];
```

But (personally), I would prefer that this checks for whitespace something like (`return !!value?.trim();`)

Reply:
Above comments are not a blocker, just a refinement

Reply:
Clean, I like it!